libs/langchain-community/src/vectorstores/chroma.ts

```diff
@@ -67,12 +73,16 @@ export class Chroma extends VectorStore {
     return "chroma";
   }
 
-  constructor(embeddings: EmbeddingsInterface, args: ChromaLibArgs) {
+  constructor(
+    embeddings: EmbeddingsInterface,
+    args: ChromaLibArgs & { clientParams?: Omit<ChromaClientParams, "path"> }
```

Comment:
Could you instead make this a single exported interface `ChromaArgs`

Reply:
Actually, just update `ChromaLibArgs` to include this `clientParams` field in both options of the type. That should be enough